test/Extensions/TesterAdoNet/RelationalUtilities/RelationalStorageForTesting.cs

```diff
@@ -114,6 +127,8 @@ public static async Task<RelationalStorageForTesting> SetupInstance(string invar
             // Concatenate scripts
             foreach (var fileName in testStorage.SetupSqlScriptFileNames)
             {
+                Console.WriteLine($"Processing {fileName}");
```

Comment:
We should either delete this or make it use `ITestOutputHelper` (an xUnit concept for logging during tests)

Reply:
So I tried to do that, then I got to that method which is static and used in a few places.

The static helper class, `TestContext`,  which exposes `WriteLine` was introduced via https://github.com/xunit/xunit/commit/850587bdbcbc0ea9e5a62a131787bd902332e747 which is xUnit and we are on 2.4.1.

<img width="367" alt="image" src="https://user-images.githubusercontent.com/234704/185034690-18ebf3d0-6e0d-4eda-a82e-90054bee4eea.png">

If you're fine with it, I can pass `ITextOutputHelper` from the callers as a parameter, but that feels dirty. 

Is there anything stopping us from upgrading xunit? I imagine lots of breaking changes or you'd had done it already.

If upgrading is not an option, I can make our own `TestContext` :)

Reply:
Passing it as a parameter or deleting the line are both fine options to me

Reply:
Upgrading xUnit is fine, too

Reply:
note: there are more than just that one `Console.WriteLine` in that method, sooo all or none. I'll figure it out.

Reply:
xunit.v3 isn't a thing yet apparently.

Yeah, that didn't go well. you can see the commit I made with the changes to remove `Console.WriteLine`. I'm not familiar enough with xUnit to know how to fix this. If you know, happy to re-revert and do it.

I just removed the offending line.